build/configs/default_config.yaml

```diff
@@ -55,6 +56,23 @@ networks:
       anchorRepository: "0x444f649e307442e76ccf737466e52f1609b98260"
       paymentObligation: "0x0417eb37941164368401D666984cED7694ABcBb1"
 
+  # Ropsten test network
+  doloresheights:
```

Comment:
This neighborhood is not as well known 😄 I would use something like `dogpatch` instead

Reply:
I liked the name .. but I can change